pages/wishlist-page/wishlist.css

```diff
@@ -0,0 +1,22 @@
+.product-container{
+    display: block;
+    flex-grow: 1;
+    margin: 1rem;
+}
+
+.product-wrapper .heading-3{
+    text-align: center;
+    font-size: var(--heading-2);
+    font-weight: bold;
+    line-height: var(--line-height-1);
+}
+
+.product-wrapper{
+    margin: auto;
+    max-width: 80%;
+}
+
+.product-menu{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(270px, 1fr));
```

Comment:
You can use `rem` unit here.

Reply:
I think, it's good to use fixed size value like px instead of rem for break point. Please correct me if I'm wrong.

Reply:
ok will use it.

Reply:
> I think, it's good to use fixed size value like px instead of rem for break point. Please correct me if I'm wrong.

The issue with px is that it's static. If the root font size is changed, this will cause an issue. Use rems just in case.

Reply:
Okay, thank you Rakshith.